docs/run-wes-example.json

```diff
@@ -13,6 +13,6 @@
         "engine": "nextflow",
         "execute_loc": "azure"
     },
-    "workflow_url": "https://github.com/phac-nml/iridanextexample",
+    "workflow_url": "https://github.com/phac-nml/clustersplitter",
```

Comment:
Given that the WES execution has changed away from a JSON file, could you remove this entire file?

Reply:
Removed in 3357071637ddac104e7159bdc0d794a725c1272a